rust/src/lib.rs

```diff
@@ -1,7 +1,13 @@
+use lazy_static::lazy_static;
 use pyo3::prelude::*;
+use pyo3_log::ResetHandle;
 
 pub mod push;
 
+lazy_static! {
+    static ref LOGGING_HANDLE: ResetHandle = pyo3_log::init();
+}
```

Comment:
I have no idea if this is safe / smart to do.

Reply:
I think this is fine! The Rust module is loaded after Python is started up, so can't see that being a problem.

And unless you write `unsafe`, I generally trust the Rust compiler to know what it's talking about with thread safety — it looks like `reset` takes `(&self)` and you're not mutating this afterwards, plus the Python GIL means it's all going to be single-threaded anyway ... I can't see why this should be concerning.